vendor/modules.txt

```diff
@@ -1287,7 +1287,6 @@ k8s.io/apiserver/pkg/apis/config
 k8s.io/apiserver/pkg/apis/config/v1
 k8s.io/apiserver/pkg/apis/config/validation
 k8s.io/apiserver/pkg/apis/example
-k8s.io/apiserver/pkg/apis/example/v1
```

Comment:
strange that this goes away with the PR.

Reply:
I got this change from -dependency test.
Let's see if there is warning this time.